src/cloud/lib/hooks/sidebar/useCloudSidebarTree.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useCallback, useMemo } from 'react'
 import {
-  mdiApplicationCog,
+  mdiCog,
```

Comment:
Delete this line, duplicate with line 14

Reply:
Fixed